cmd/config_validation_test.go

```diff
@@ -46,6 +47,19 @@ func getConfigObjectWithConfigFile(t *testing.T, configFilePath string) (*cfg.Co
 	return getConfigObject(t, []string{fmt.Sprintf("--config-file=%s", configFilePath)})
 }
 
+func defaultFileCacheConfig(t *testing.T) cfg.FileCacheConfig {
+	t.Helper()
+	return cfg.FileCacheConfig{
+		CacheFileForRangeRead:    false,
+		DownloadChunkSizeMb:      50,
+		EnableCrc:                false,
+		EnableParallelDownloads:  false,
+		MaxParallelDownloads:     int64(max(16, 2*runtime.NumCPU())),
```

Comment:
Use a literal
```suggestion
		MaxParallelDownloads:    5,
```

Reply:
I don't think we can use a literal here as we're checking the default config values in this test: 
`TestValidateConfigFile_FileCacheConfigSuccessful - Empty config file [default values].`